internal/commands/test.go

```diff
@@ -144,19 +144,23 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 
 // GetResult returns the result of testing the structured data against their policies
 func GetResult(ctx context.Context, namespace string, input interface{}, compiler *ast.Compiler) (CheckResult, error) {
```

Comment:
We've changed the behavior here in that the result will now have a successes slice populated
Perhaps we should stress that functionality in a test for
`GetResult` in `﻿/internal/commands/test_test.go`
??

Reply:
I added some test.